lib/games_collection.rb

```diff
@@ -369,4 +369,32 @@ def best_defense_id
   def worst_defense_id
     average_goals_scored_by_opposite_team.max_by { |team_id, goals| goals }.first
   end
+
+  def biggest_team_blowout_num(teamid)
+    difference_array  = games.reduce([]) do |acc, game|
```

Comment:
look at creating a new method for difference array so you can use it for worse loss as well.